hazelcast/src/test/java/com/hazelcast/test/HazelcastParallelClassRunner.java

```diff
@@ -149,12 +149,7 @@ public void run() {
 
         private final Properties globalProperties;
 
-        private final ThreadLocal<Properties> localProperties = new InheritableThreadLocal<Properties>() {
-            @Override
-            protected Properties initialValue() {
-                return init(new Properties());
-            }
-        };
+        private final ThreadLocal<Properties> localProperties = new ThreadLocal<Properties>();
```

Comment:
I changed `new InheritableThreadLocal()` to `new ThreadLocal()` by accident, that caused the test failure.